packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -141,10 +144,11 @@ export class OdspDelayLoadedDeltaStream {
 			// websocket token or whether it is returned with joinSession response payload
 			const requestWebsocketTokenFromJoinSession = this.getWebsocketToken === undefined;
 			const websocketTokenPromise = requestWebsocketTokenFromJoinSession
-				? Promise.resolve(null)
+				? // eslint-disable-next-line unicorn/no-null
+				  Promise.resolve(null)
```

Comment:
o_O. Can this be like the suggestion or does the compiler want the `?` immediately after the condition?

```suggestion
				// eslint-disable-next-line unicorn/no-null
				? Promise.resolve(null)
```

Reply:
I tried that, and prettier put it back to this :(